pkg/webhooks/mutator/virt-launcher-mutator.go

```diff
@@ -0,0 +1,178 @@
+package mutator
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"strconv"
+
+	"k8s.io/apimachinery/pkg/api/resource"
+
+	"github.com/go-logr/logr"
+	"gomodules.xyz/jsonpatch/v2"
+	admissionv1 "k8s.io/api/admission/v1"
+	k8sv1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/equality"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/log"
+	"sigs.k8s.io/controller-runtime/pkg/webhook/admission"
+
+	"github.com/kubevirt/hyperconverged-cluster-operator/api/v1beta1"
+)
+
+// TODO: Is that needed?
```

Comment:
This is a way to promise in compile time, that the `VirtLauncherMutator` type actually implements the `admission.Handler` interface.

Reply:
Makes sense! Thanks a lot!